src/adapters/mc/OsConfigPolicy_DeployIfNotExists.json

```diff
@@ -38,6 +54,46 @@
                 ],
                 "defaultValue": "false"
             },
+            "AccessPermissionsForSshdConfig": {
+                "type": "string",
+                "metadata": {
+                    "displayName": "Access permissions for sshd_config",
+                    "description": "File access permissions for /etc/ssh/sshd_config. Default is '600'"
+                },
+                "defaultValue": "600"
+            },
+            "Protocol": {
+                "type": "string",
+                "metadata": {
+                    "displayName": "Protocol",
+                    "description": "The SSH protocol for sshd. Default is '2'"
```

Comment:
It seems redundant to mention what the default is.

Reply:
This is by-design from AzP side. This policy definition is generated based on inputs we give there which are not checked in but can be seen here. The names and descriptions and the default values are ours for most part, rest isn't.

Reply:
I mean to mention the default in the description. That's already implied by property below (and rendered by UI).